projects/vdk-core/tests/functional/ingestion/utils.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pathlib
+
+from vdk.plugin.test_utils.util_funcs import get_test_job_path
+
+
+def job_path(job_name: str) -> str:
+    return get_test_job_path(
```

Comment:
If you think get_test_job_path could be simplified, have you considered contributing to vdk-test-utils to reuse everywhere?

Reply:
Will refactor the method and maybe add it to vdk-test-utils in a separate PR. Thanks